src/util/config/index.js

```diff
@@ -213,13 +216,24 @@ config.set(
 
 config.set('isProd', config.get('env') === 'production');
 config.set('isDev', config.get('env') === 'development');
-config.validate();
 const appConf = config.get('app_server');
 if (
 	appConf.protocol === 'https' &&
 	(!fs.existsSync(appConf.key_file) || !fs.existsSync(appConf.crt_file))
 ) {
-	throw new Error('Missing HTTPS cert or key!');
+	const message = 'Missing HTTPS cert or key for application server!';
+	if (config.isProd) {
```

Comment:
I think this could break prod.

RIght now we're not including certs in the app server, only in the configs for the nginx server. This was OK before, because `http` was the default for production. If we're definitely overwriting this value in prod, it's OK, but I'm not sure if that's the case.